doc-content/jbpm-docs/src/main/asciidoc/BPMN2/create-service-task-proc.adoc

```diff
@@ -46,18 +76,106 @@ A service task is a task that is part of the process that is executed outside of
 | *Is Multiple Instance*
 | Select if this task has multiple instances.
 
+| *MI Execution mode*
+| Select if the multiple instances execute in parallel or sequentially.
+
+| *MI Collection input*
+| Specify a variable that represents a collection of elements for which new instances are created, such as `inputCountryNames`.
+
+| *MI Data Input*
+| Specify the input data assignment that is transferred to a web service, such as `Parameter`.
+
+| *MI Collection output*
+| The array list in which values returned from the web service task is stored, such as `outputCountries`.
+
+| *MI Data Output*
+| Specify the output data assignment for the web service task, which stores the result of class execution on the server, such as `Result`.
+
+| *MI Completion Condition (mvel)*
+| Specify the MVEL expression that is evaluated on each completed instance to check if the specified multiple instance node can complete.
+
 | *On Entry Action*
 | A Java, JavaScript, or MVEL script that specifies an action at the start of the task.
 
 | *On Exit Action*
 | A Java, JavaScript, or MVEL script that specifies an action at the end of the task.
 
-
 | *SLA Due Date*
 | The date that the service level agreement (SLA) expires.
 
 
 |===
 
+.Example of creating Java service task
+When you create a service task using Java method, then the method can only contain one parameter and returns a single value. To create a service task using a Java method, you must add the Java class to the dependencies of the project:
+
+.. Create a business process.
+.. In the process designer, select the *Activities* tool from the tool palette.
+.. Select *Service Task*.
+.. Click a blank area of the process designer canvas.
+.. If necessary, in the upper-right corner of the screen, click the *Properties* icon.
+.. Add or define the task information listed in the following table as required.
++
+.Java service task parameters
+[cols="30%,70%", options="header"]
+|===
+|Label
+|Description
+
+| *Name*
+| The name of the service task. You can also double-click the shape to edit the name of the service task.
+
+| *Documentation*
+| Enter a description of the task. The text in this field is included in the process documentation. Click the *Documentation* tab in the upper-left side of the process designer canvas to view the process documentation.
+
+| *Implementation*
+| Specify the task is implemented in Java.
 
+| *Interface*
+| The class used to implement the script, such as `org.xyz.HelloWorld`.
+
+| *Operation*
+| The method that will be called by the interface, such as `sayHello()`.
```

Comment:
I believe this should be without parentheses.

Reply:
Since it is not a normal text, that is the reason we add parantheses.

Reply:
Yes, but a user should put there `sayHello` when designing a process, so it can be misleading.